code/common/finspace.q

```diff
@@ -55,3 +55,11 @@ newrdbup:{[]
       .lg.o[`newrdbup;"received signal from next period rdb, setting rdbready to true"];
       @[`.finspace;`rdbready;:;1b];
  };
+
+ // clustername must be of type string
```

Comment:
should we add a check for this & throw a descriptive error if it isn't a string?

Reply:
added the following: `if[10h<>fType:type clustername; .lg.o[`deletecluster;"clustername must be of type string: 10h, got ",-3!fType]; :(::)]; `